---
avatar_filename: ""
bio: My current research interests span a broad range of molecular and population genetic processes, involving the study of patterns of genetic diversity and molecular evolution at the scale of genomes, species, and populations. It includes the causes and consequences of meiotic recombination, the evolution of gene expression and genetic structural variation among populations and species.
interests:
  - meiotic recombination and genetic shuffling
  - genomic patterns of genetic diversity
  - evolution of gene expression
  - genomic structural variation
social:
  - icon: envelope
    icon_pack: fas
    link: /#contact
  - icon: mastodon
    icon_pack: fab
    link: https://ecoevo.social/@thomasbrazier
    label: Follow me on Mastodon
    display:
      header: true
  - icon: google-scholar
    icon_pack: fab
    link: https://scholar.google.com/citations?hl=fr&user=iMW4DGQAAAAJ
  - display:
      header: false
    link: "https://orcid.org/0000-0001-5990-7545 "
    icon: orcid
    label: ORCID
    icon_pack: fab
  - icon: github
    icon_pack: fab
    link: https://github.com/ThomasBrazier
  - icon: linkedin
    icon_pack: fab
    link: https://www.linkedin.com/in/thomas-brazier-0b2108256/
organizations:
  - name: University of Rennes 1 - ECOBIO Lab UMR 6553
    url: https://ecobio.univ-rennes1.fr/
education:
  courses:
    - course: PhD in Ecology & Evolution
      institution: University of Rennes 1
      year: 2022
    - course: Master's Degree in Ecology & Evolution with a major in Modelling, with
        honors, first of the year
      institution: University of Rennes 1
      year: 2019
    - course: BSc in Life Sciences and Biology of Organisms, with honors, first of the
        year
      institution: Pierre and Marie Curie Paris 6 University and Aix-Marseille University
      year: 2017
superuser: true
last_name: Brazier
role: Post-doc in Population Genomics and Evolutionary Biology
highlight_name: true
title: Thomas Brazier
first_name: Thomas
email: "brazier.thomas@gmail.com"
---
I am an evolutionary biologist and population geneticist broadly interested in genome evolution and patterns of genetic diversity across genomes, populations, and species. My academic background combines a PhD in population genomics focused on recombination landscapes and ongoing research in comparative genomics and genome evolution, with a focus on understanding the diversity of structural variation across the eukaryotic tree of life. My work has allowed me to gain experience with genomic data analysis, including variant calling and genotyping, estimating genetic diversity statistics, simulations under diverse frameworks (msprime, SLiM), and Python/R programming in Unix environments. I have also actively engaged in analyzing large-scale genomic and life history trait datasets to understand evolutionary processes, such as the evolution of recombination landscapes across plant species.


During my PhD ([ANR HotRec](https://anr.fr/Project-ANR-19-CE12-0019)), I leveraged large genomic datasets in a comparative genomic framework to understand global patterns of molecular evolution in plants. I used meta-analytic approaches and advanced statistics to describe and quantify the diversity and determinants of recombination and its associated evolutionary processes (e.g., GC-biased gene conversion) among a large set of plant species.


After defending my PhD thesis in late 2022, I was rapidly involved in a wide range of themes spanning molecular evolution, population genomics, and evolutionary biology. My first postdoctoral position focused on testing an original theory of the evolution of gene expression (the runaway process proposed by Thomas Lenormand, [ANR CisTransEvol](https://anr.fr/Project-ANR-22-CE02-0024)). More recently, I have involved myself in the newly arising field of pangenomics and genomic structural variation ([ERC](https://cordis.europa.eu/project/id/101115983) funded, supervised by Claire MÃ©rot). Among these widespread spectrum of topics, I have taken a broad view of population genomics, studying evolution at various levels of organization, including genes, genomes, populations, and species. I am interested in integrative approaches that reconcile different levels of organization under a more general and synthetic model (e.g., how patterns at the scale of genes could influence global genome evolution, and how both are co-evolving).


<!-- 
My new project/future directions
 -->


I am also a continuous learner, very curious and eager to train myself in a broad range of topics and methods in evolutionary genomics. I am involved in the development of bio-informatic pipelines and packages ([sv calling](), [recombination maps]()) as well as the development of modern statistical methods for population genomics, especially Bayesian inference, Machine Learning and Neural Netwrosk. In particular, I am actively developping Bayesian neural networks methods for inference of demographic parameters in an Approximate Baysian Computation framework. As Deep Learning is getting more and more applications in population genomics, I believe it is a powerful versatile approach for inferences and predictions on large scale genomic data with complex underlying models and non-linear relationships in high-dimension datasets.

{{< icon name="download" pack="fas" >}} Download my {{< staticref "uploads/resume.pdf" "newtab" >}}resume{{< /staticref >}}
