---
avatar_filename: ""
bio: My current research interests span a broad range of molecular and population genetic processes, involving the study of evolutionary patterns at the scale of genomes, species, and populations. It includes the causes and consequences of meiotic recombination, the evolution of gene expression and genome structural variation among populations and species.
interests:
  - meiotic recombination and genetic shuffling
  - genomic patterns of genetic diversity
  - evolution of gene expression
  - genomic structural variation
social:
  - icon: envelope
    icon_pack: fas
    link: /#contact
  - icon: mastodon
    icon_pack: fab
    link: https://ecoevo.social/@thomasbrazier
    label: Follow me on Mastodon
    display:
      header: true
  - icon: google-scholar
    icon_pack: fab
    link: https://scholar.google.com/citations?hl=fr&user=iMW4DGQAAAAJ
  - display:
      header: false
    link: "https://orcid.org/0000-0001-5990-7545 "
    icon: orcid
    label: ORCID
    icon_pack: fab
  - icon: github
    icon_pack: fab
    link: https://github.com/ThomasBrazier
  - icon: linkedin
    icon_pack: fab
    link: https://www.linkedin.com/in/thomas-brazier-0b2108256/
organizations:
  - name: University of Rennes 1 - ECOBIO Lab UMR 6553
    url: https://ecobio.univ-rennes1.fr/
education:
  courses:
    - course: PhD in Ecology & Evolution
      institution: University of Rennes 1
      year: 2022
    - course: Master's Degree in Ecology & Evolution with a major in Modelling, with
        honors, first of the year
      institution: University of Rennes 1
      year: 2019
    - course: BSc in Life Sciences and Biology of Organisms, with honors, first of the
        year
      institution: Pierre and Marie Curie Paris 6 University and Aix-Marseille University
      year: 2017
superuser: true
last_name: Brazier
role: Post-doc in Genomic and Evolution
highlight_name: true
title: Thomas Brazier
first_name: Thomas
email: ""
---
I am an evolutionary biologist broadly interested in genome evolution and patterns of genetic diversity. I primarily use large genomic datasets (based on already published studies) to re-analyse them in a novel comparative genomic framework and discover global patterns of molecular evolution. During my PhD (ANR HotRec) I used meta-analytic approaches and advanced statistics to describe and quantify the diversity and determinants of evolutionary processes (e.g. recombination, GC-biased gene conversion) among a large set of species.

Though I am still a young researcher (PhD defended in late 2022), I was rapidly involved in a wide range of thematics spanning molecular evolution, population genomics and evolutionary biology at a broad sense. My first post-doc is actually about testing an original theory of evolution of gene expression (the runaway process proposed by Thomas Lenormand, ANR CisTransEvol), while I am at the same time involved in the newly arising field of pangenomic and structural variation within genomes (ERC funded, supervised by Claire MÃ©rot). Among these very different topics I find it very interesting to study evolution at many levels of organisation, including genes, genomes, populations and species. I am interested in integrative approaches reconciling different levels of organisation under a more general and synthetic model (e.g. how patterns at the scale of genes could influence global genome evolution, how both are co-evolving).

Alternatively, I am also involved in the development of bio-informatic pipelines as well as modern statistical methods for population genomics, especially Artificial Intelligence. As Deep Learning is getting more and more applications, I believe it is a powerful versatile approach for population genomics. Deep Learning, especially Neural Networks, has proven to be "unreasonably effective" for inferences and predictions on large scale genomic data and almost any problem can be adressed successfuly with Neural Networks.

{{< icon name="download" pack="fas" >}} Download my {{< staticref "uploads/resume.pdf" "newtab" >}}resume{{< /staticref >}}
